pioneer/packages/joy-tokenomics/src/Overview/SpendingAndStakeDistributionTable.tsx

```diff
@@ -229,6 +232,30 @@ const SpendingAndStakeDistributionTable: React.FC<{data?: TokenomicsData; status
           stakeShare={data && `${round(data.contentCurators.lead.stakeShare * 100)}`}
           color='rgb(100, 160, 190)'
         />
+        <SpendingAndStakeTableRow
+          role='Operations'
+          helpContent='The current Operations members, and the sum of their projected rewards and stakes.'
+          numberOfActors={data && `${data.operations.number}`}
+          groupEarning={data && `${Math.round(data.operations.rewardsPerWeek)}`}
+          groupEarningDollar={displayStatusData('operations', 'rewardsPerWeek')}
+          earningShare={data && `${round(data.operations.rewardsShare * 100)}`}
+          groupStake={data && `${data.operations.totalStake}`}
+          groupStakeDollar={displayStatusData('operations', 'totalStake')}
+          stakeShare={data && `${round(data.operations.stakeShare * 100)}`}
+          color='rgb(30, 166, 165)'
+        />
+        <SpendingAndStakeTableRow
+          role='Operations Lead'
+          helpContent='Current Operations Lead, and their projected reward and stake.'
+          numberOfActors={data && `${data.operations.lead.number}`}
+          groupEarning={data && `${Math.round(data.operations.lead.rewardsPerWeek)}`}
+          groupEarningDollar={displayStatusData('operations', 'rewardsPerWeek', true)}
+          earningShare={data && `${round(data.operations.lead.rewardsShare * 100)}`}
+          groupStake={data && `${data.operations.lead.totalStake}`}
+          groupStakeDollar={displayStatusData('operations', 'totalStake', true)}
+          stakeShare={data && `${round(data.operations.lead.stakeShare * 100)}`}
+          color='rgb(30, 132, 189)'
```

Comment:
I noticed we're using the same rgb values here and in `TokenomicsCharts.tsx`.
It's not a priority, but it would be nice to have those colors represented by shared (exported) constants like `OPERATIONS_CHART_COLOR`, `OPERATIONS_LEAD_CHART_COLOR`, `CURATORS_CHART_COLOR` etc. (or maybe `CHART_COLORS.OPERATIONS`, `CHART_COLORS.OPERATIONS_LEAD` etc.) to avoid "magic values" which may lead to potential mismatches.